src/Symfony/Component/HttpFoundation/Tests/ServerBagTest.php

```diff
@@ -177,4 +177,20 @@ public function testItDoesNotOverwriteTheAuthorizationHeaderIfItIsAlreadySet()
             'PHP_AUTH_PW' => '',
         ], $bag->getHeaders());
     }
+
+    /**
+     * An HTTP request without content-type and content-length will result in
+     * the variables $_SERVER['CONTENT_TYPE'] and $_SERVER['CONTENT_LENGTH']
+     * containing an empty string in PHP.
+     */
+    public function testRequestWithoutContentTypeAndContentLength()
+    {
+        $bag = new ServerBag([
+            'CONTENT_TYPE' => '',
+            'CONTENT_LENGTH' => '',
```

Comment:
Content md5 should be added to the testcase

Reply:
If the request does not contain 'Content-MD5', PHP does not set $_SERVER['CONTENT_MD5'] to an empty string, unlike 'Content-Type' and 'Content-Length'. So, I don't think it is necessary.

Reply:
Good to know, thank you